shared-data/pipette/schemas/2/pipetteGeometrySchema.json

```
@@ -32,6 +32,14 @@
       "description": "The path to a valid Opentrons shared schema relative to the shared-data directory, without its extension. For instance, #/pipette/schemas/2/pipetteGeometrySchema.json is a reference to this schema."
     },
     "nozzleOffset": { "$ref": "#/definitions/xyzArray" },
+    "pipetteBoundingBoxOffsets": {
+      "description": "Offsets from the pipette mount position to the back-left & front-right corners of the pipette's casing around the nozzles.",
```

From your numbers, it looks like you're wrapping the bounding box around the casing *and* the nozzles, but this description makes it sound like it's supposed to just wrap around the casing.

Totally up to you how it should be, but if it's intended to wrap around both the casing *and* the nozzles, here's a suggested rewording:

```suggestion
      "description": "Offsets from the pipette mount position to the back-left & front-right corners of the bounding box containing the whole pipette, including its nozzles and casing.",
```